howto-gallery/package.json

```diff
@@ -5,25 +5,53 @@
   "description": "Minecraft TypeScript How-to Gallery",
   "private": true,
   "devDependencies": {
-    "del": "^6.0.0",
-    "gulp": "^4.0.2",
-    "gulp-cli": "^2.3.0",
-    "gulp-sourcemaps": "^3.0.0",
-    "gulp-typescript": "^6.0.0-alpha.1",
+    "@minecraft/core-build-tasks": "../resources/minecraft-core-build-tasks-1.0.2-patch-34.tgz",
```

Comment:
don't forget to update upon publish

Reply:
yup, and also re run npm i to fix .lock files too